apps/web/components/TrackPreview.tsx

```diff
@@ -12,39 +11,20 @@ type TrackPreviewProps = {
   track: any;
 };
 
-const truncateDescription = (text: string, wordLimit: number) => {
```

Comment:
Why did u remove all this code?

Reply:
This code is basically checking width of screen size in , every time screen size changes and based on that it truncate the description of tracks . I removed this because now it have scroll functionality so we dont have to truncate the description of tracks and i fixed the height of the div tracks description in so that it wont take all the height